Sources/Rendering/Core/Mapper/CoincidentTopologyHelper.js

```diff
@@ -53,6 +69,8 @@ function implementCoincidentTopologyMethods(publicAPI, model) {
     publicAPI[methodName] = staticOffsetAPI[methodName];
   });
 
+  macro.setGet(publicAPI, model, ['resolveCoincidentTopology']);
+
```

Comment:
This is an interesting change. The static methods declare `vtkMapper.setResolveCoincidentToplogy(mode = 0)`, while the instance methods are used as a boolean value (e.g. `mapper.setResolveCoincidentTopology(true)`). Should we change expectations of the latter to match the former?

Reply:
I think `mapper.setResolveCoincidentTopology(true)` should not be allowed.

Reply:
I agree with @finetjul. The setter should just be setting the type of resolution.